$OpenBSD: patch-src_game_cpp,v 1.3 2011/07/08 19:14:59 jasper Exp $

Prevent compilation error on arm:
game.cpp:1307: warning: `int res' might be used uninitialized in this function

--- src/game.cpp.orig	Sat May  7 17:11:50 2011
+++ src/game.cpp	Fri Jul  8 19:58:29 2011
@@ -1150,7 +1150,7 @@ void game_controller::start_wesnothd()
 
 bool game_controller::play_multiplayer()
 {
-	int res;
+	int res = 0;
 
 	state_ = game_state();
 	state_.classification().campaign_type = "multiplayer";
